podcasts/ProfileViewController.swift

```diff
@@ -240,11 +241,26 @@ class ProfileViewController: PCViewController, UITableViewDataSource, UITableVie
             return tableView.dequeueReusableCell(withIdentifier: endOfYearPromptCell, for: indexPath) as! EndOfYearPromptCell
         }
 
+        if FeatureFlag.kidsProfile.enabled && row == .kidsProfile {
```

Comment:
I normally prefer to use `,` instead of `&&` in Swift conditions. What do you think?

Reply:
No preferences at all 👍